packages/experimental/Avatar/src/useAvatar.ts

```diff
@@ -39,6 +39,7 @@ export const useAvatar = (props: JSAvatarProps): AvatarInfo => {
 
   return {
     props: {
+      accessible: true,
```

Comment:
We should also make this `accessible: accessible ?? true`

Reply:
> We should also make this `accessible: accessible ?? true`

But in this case if `accessible` is set to false we'll always get true

Reply:
I don't think it should. The `??` operator will only return the right-hand side when the left-hand side side is null or undefined but if it has a value (even if that value is `false`) it should return the left-hand side.

Reply:
You're right. I don't get why it didn't work the first time I tried it